app/src/main/kotlin/at/bitfire/davdroid/ui/intro/PermissionsIntroPage.kt

```diff
@@ -31,4 +31,9 @@ class PermissionsIntroPage: IntroPage {
         PermissionsContent()
```

Comment:
This line creates a new model by the default argument `model = viewModel()`, which is then a different model as used in `onResume`.